wpilibNewCommands/src/main/native/include/frc2/command/SelectCommand.h

```diff
@@ -76,6 +77,7 @@ class SelectCommand : public CommandHelper<Command, SelectCommand<Key>> {
       : m_selector{std::move(selector)} {
     for (auto&& command : commands) {
       CommandScheduler::GetInstance().RequireUngrouped(command.second.get());
+      command.second.get()->SetComposed(true);
```

Comment:
This wasn't done before?
This is a bug that needs to be fixed, but I think it's out of scope for this PR.

Reply:
I can move it into a separate PR, but yeah that surprised me too.